### DISCUSSION TITLE
Quick Capture creating a New Note - User inputs content and new note title

### DISCUSSION THREAD

**Discussion by GitMurf** [SilentVoid13/Templater#231] [Templates Showcase]
Code: https://gist.github.com/GitMurf/292ea4e17e36e0910c191c1e5b39145f

This script is a spin off of the Quick Capture script. The different is that it will create a new note each time. A modal input will pop up asking for the content you want to add. And then a second modal input will pop up for you to enter the note title. It defaults to the root of your vault, but you can update the `let folderOverride = '/';` variable if you want to specify a different location.

Discussion on Discord here: https://discord.com/channels/686053708261228577/840286238928797736/845425365551415297

**Comment by thatch11** [comment]
This is brilliant! Question, is it possible to add some code to then open the new note immediately?

**Reply by joschsch** [reply]
It would defeat the purpose - if you want to work with a new note, just ctrl+shift+n would already work. This helps if you want to just have a quick note done somewhere else and maybe work on it later.
